### PR TITLE
river-tag-overlay: init  at 1.0.0

### PR DESCRIPTION
**Pull request by edrex**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
- [x] Tested, as applicable:
  - no pkg tests
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/river-tag-overlay/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  makeFlags = [
+    "DESTDIR=${placeholder ''out''}"
+    "PREFIX="
+  ];
+
+  meta = with lib; {
+    description = "A pop-up showing tag status";
+    homepage = "https://git.sr.ht/~leon_plickat/lswt";
+    license = licenses.gpl3;
+    maintainers = [ maintainers.edrex ];
```
Comment (rodrgz) [inline pkgs/applications/misc/river-tag-overlay/default.nix:25]: It seems that you need to add yourself to `maintainers-list.nix` file.

**Review by rodrgz — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/river-tag-overlay/default.nix (hunk 1 above)
Comment (edrex) [reply pkgs/applications/misc/river-tag-overlay/default.nix:25]: Done, and I broken out the maintainer-list.nix change into a separate commit that's shared with #158529 as suggested by @urandom2.

**Review by rodrgz — APPROVED** [review]
LGTM

File: pkgs/applications/misc/river-tag-overlay/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  version = "1.0.0";
+
+  src = fetchFromSourcehut {
+    owner = "~leon_plickat";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-hLyXdLi/ldvwPJ1oQQsH5wgflQJuXu6vhYw/qdKAV9E=";
+  };
+
+  buildInputs = [ wayland pixman pkg-config ];
```
Comment (urandom2) [inline pkgs/applications/misc/river-tag-overlay/default.nix:14]: nit: can you sort this
```suggestion
  buildInputs = [ pixman pkg-config wayland ];
```

**Review by urandom2 — APPROVED** [review]
if ofborg is happy, so am I

**Comment by edrex** [comment]
is it ok to merge a package that doesn't build on aarch64-linux? It should eventually be fixed upstream.

**Comment by urandom2** [comment]
yes, but we should flag it as broken by adding `meta.broken = stdenv.isAarch64;`

**Comment by edrex** [comment]
ok, marked as broken on `Aarch64`. I think this is ready. Thanks for all the help with review @urandom2

**Review by urandom2 — APPROVED** [review]

File: pkgs/applications/misc/river-tag-overlay/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  version = "1.0.0";
+
+  src = fetchFromSourcehut {
+    owner = "~leon_plickat";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-hLyXdLi/ldvwPJ1oQQsH5wgflQJuXu6vhYw/qdKAV9E=";
+  };
+
+  buildInputs = [ pixman pkg-config wayland ];
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/river-tag-overlay/default.nix:14]: pkg-config goes to nativeBuildInputs otherwise cross compilation will not work.

**Review by rodrgz — APPROVED** [review]

**Review by thiagokokada — APPROVED** [review]

**Pull request merged**